app/scripts/TiledPlot.js

```diff
@@ -1030,7 +1030,7 @@ class TiledPlot extends React.Component {
             width: this.leftWidth + this.centerWidth + this.rightWidth,
             height: this.topHeight + this.centerHeight
             + this.bottomHeight
-            + 2 * this.props.verticalMargin,
+            + this.props.paddingTop + this.props.paddingBottom,
```

Comment:
This is going to break things. Margin and padding are not the same. What is the reason for changing this?